README.rst

```diff
@@ -18,7 +18,9 @@ SaltPad is mainly coded in Javascript and should be compatible with all modern b
 
 SaltPad communicate with Salt through the salt-api and thus requires access to the salt-api from the browser. If this is an issue, please drop a comment on [this issue](http://github.com/tinyclues/saltpad) to discuss the possible solutions. The salt-api format / specification is not yet stable, so SaltPad could only provide limited compatibility with salt-api. The salt-api format depends on 3 variables, salt version, the netapi used (cherrypy or tornado) and the master_job_cache used for storing and retrieving jobs results.
 
-**SaltPad required some upgrade on salt-api side (for CORS support mainly) and will only work with a develop version of saltstack.**
+**SaltPad required some upgrade on salt-api side (for CORS support mainly) and will only work with a develop version of saltstack if deployed apart.**
+
+**SaltPad could be deployed via rest_cherrypy (see Rest_cherrypy single-app page deployment part below) and should works with a stable version but consider it beta for the moment.**
```

Comment:
SaltPad depends on some upgrades in salt-api (for CORS support mainly) and will only work with a develop version of saltstack until those upgrades will be merged in a stable release.

Otherwise, SaltPad can use the rest_cherrypy API (and so, a stable version of saltstack), but this should still be considered in beta. See Rest_cherrypy single-app page deployment for more infos.